lib/publiccloud/ssh_interactive.pm

```diff
@@ -144,6 +144,7 @@ sub prepare_ssh_tunnel {
 
     # Permit root passwordless login over SSH
     $instance->ssh_assert_script_run('sudo sed -i "s/PermitRootLogin no/PermitRootLogin prohibit-password/g" /etc/ssh/sshd_config');
+    $instance->ssh_assert_script_run('sudo sed -i "s/#AllowTcpForwarding yes/AllowTcpForwarding yes/g" /etc/ssh/sshd_config');
```

Comment:
can we limit this to hardended images only ?

Reply:
Why? This is a necessary requirement for our test runs everywhere and the usage of `sed` only alters this if needed.

Reply:
because currently this requirement is coming from image . and without this change being merged when image for whatever reason change this our test will fail and investigation of this failure will end up on product bug. If we will accept this change as is than such change in image will pass through our tests not noticed

Reply:
Yes, that makes sense - Let's limit the change to hardened images only!